corehq/apps/app_manager/models.py

```diff
@@ -818,6 +818,7 @@ class FormBase(DocumentSchema):
     form_links = SchemaListProperty(FormLink)
     schedule_form_id = StringProperty()
     custom_instances = SchemaListProperty(CustomInstance)
+    case_references_data = DictProperty()
```

Comment:
decided to take a crack at this in most recent changes. it's not perfect because the current structure of each question being a key doesn't lend itself well to being typed, but I still think it'll be better to work with.